### PR TITLE
add settings bar service for local storege

### PR DESCRIPTION
**Pull request by mrSelieznov**
#175  Task 

Add new variable to local storage to save setting of history bur

### CODE REVIEW

File: src/models/index.tsx [hunk 1]
```diff
@@ -17,6 +17,12 @@ export enum Theme {
   dark,
   light
 }
+
+export enum HistoryBur {
```
Comment (oleksandrstarov) [inline src/models/index.tsx:21]: https://translate.google.com/?hl=ru&sl=en&tl=ru&text=Bur&op=translate

Check speling

File: src/service/api.tsx [hunk 2]
```diff
@@ -1,16 +1,18 @@
 import { API_KEY, API_IMG_URL } from '../constants/api';
 import axios from '../axios/url';
 
-import { Theme, MovieCard } from '../models/index';
+import { Theme, MovieCard, HistoryBur } from '../models/index';
+
 const apiService: {
   storeKey: string;
-  store: { history: number[]; favorites: number[]; theme: any };
+  store: { history: number[]; favorites: number[]; theme: any; historyBur: any };
```
Comment (oleksandrstarov) [inline src/service/api.tsx:8]: why any?

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/service/api.tsx [hunk 3]
```diff
@@ -1,16 +1,18 @@
 import { API_KEY, API_IMG_URL } from '../constants/api';
 import axios from '../axios/url';
 
-import { Theme, MovieCard } from '../models/index';
+import { Theme, MovieCard, HistoryBar } from '../models/index';
+
 const apiService: {
   storeKey: string;
-  store: { history: number[]; favorites: number[]; theme: any };
+  store: { history: number[]; favorites: number[]; theme: number; historyBar: number };
```
Comment (oleksandrstarov) [inline src/service/api.tsx:8]: Wonder why not Theme and HistoryBar?

File: src/models/index.tsx [hunk 4]
```diff
@@ -17,6 +17,12 @@ export enum Theme {
   dark,
   light
 }
+
+export enum HistoryBar {
+  unsubscribe,
+  subscribe
+}
```
Comment (oleksandrstarov) [inline src/models/index.tsx:24]: The naming is not perfect
maybe 
enum FeatureStatus { enabled, disabled } ?

File: src/service/api.tsx [hunk 5]
```diff
@@ -28,12 +30,20 @@ const api = {
...
       favorites: [],
-      theme: 1
+      theme: 1,
+      historyBar: HistoryBar.subscribe
     }
   ) {
     localStorage.setItem(apiService.storeKey, JSON.stringify(data));
   },
 
+  switchHistoryBur() {
```
Comment (oleksandrstarov) [inline src/service/api.tsx:40]: check spelling please

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/service/api.tsx (hunk 3 above)
Comment (mrSelieznov) [reply src/service/api.tsx:8]: are you mean to make these name from capital letter?

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/service/api.tsx (hunk 3 above)
Comment (oleksandrstarov) [reply src/service/api.tsx:14]: I meant types for the values

**Comment by oleksandrstarov** [comment]
Check one last comment

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]
Check comment regarding types

**Review by oleksandrstarov — APPROVED** [review]

**Review by Kachyr — APPROVED** [review]

**Pull request merged**